src/plugins/vis_type_vega/public/vega_visualization.test.js

```diff
@@ -185,49 +185,5 @@ describe.skip('VegaVisualizations', () => {
         vegaVis.destroy();
       }
     });
-
-    test('should add a small subpixel value to the height of the canvas to avoid getting it set to 0', async () => {
-      let vegaVis;
-      try {
-        vegaVis = new VegaVisualization(domNode, vis);
-        const vegaParser = new VegaParser(
-          `{
-            "$schema": "https://vega.github.io/schema/vega/v5.json",
-            "marks": [
-              {
-                "type": "text",
-                "encode": {
-                  "update": {
-                    "text": {
-                      "value": "Test"
-                    },
-                    "align": {"value": "center"},
-                    "baseline": {"value": "middle"},
-                    "xc": {"signal": "width/2"},
-                    "yc": {"signal": "height/2"}
-                    fontSize: {value: "14"}
-                  }
-                }
-              }
-            ]
-          }`,
-          new SearchAPI({
-            search: dataPluginStart.search,
-            uiSettings: coreStart.uiSettings,
-            injectedMetadata: coreStart.injectedMetadata,
-          })
-        );
-        await vegaParser.parseAsync();
-
-        mockedWidthValue = 256;
-        mockedHeightValue = 256;
-
-        await vegaVis.render(vegaParser);
-        const vegaView = vegaVis._vegaView._view;
-        expect(vegaView.height()).toBe(250.00000001);
```

Comment:
I think problem was in that line. I decide to remove that test cause we already have have some snapshots with that adjusted height
![image](https://user-images.githubusercontent.com/20072247/87530500-143c1800-c699-11ea-8efc-26c22d22ed0b.png)